examples/vdi_profile_example.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""
+Creating power demand profiles using bdew profiles.
+
+Installation requirements
+-------------------------
+This example requires at least version v0.1.4 of the oemof demandlib. Install
+by:
+    pip install 'demandlib>=0.1.4,<0.2'
+Optional:
+    pip install matplotlib
+
+SPDX-FileCopyrightText: Uwe Krien <krien@uni-bremen.de>
+
+SPDX-License-Identifier: MIT
+
+"""
+
+import datetime
+
+from matplotlib import pyplot as plt
+
+from demandlib import vdi
+
+# The following dictionary has been created by "workalendar"
+# pip3 install workalendar
+# from workalendar.europe import Germany
+# cal = Germany()
+# holidays = dict(cal.holidays(2017))
+
+holidays = {
+    datetime.date(2017, 1, 1): "New year",
+    datetime.date(2017, 4, 14): "Good Friday",
+    datetime.date(2017, 4, 17): "Easter Monday",
+    datetime.date(2017, 5, 1): "Labour Day",
+    datetime.date(2017, 5, 25): "Ascension Thursday",
+    datetime.date(2017, 6, 5): "Whit Monday",
+    datetime.date(2017, 10, 3): "Day of German Unity",
+    datetime.date(2017, 10, 31): "Reformation Day",
+    datetime.date(2017, 12, 25): "Christmas Day",
+    datetime.date(2017, 12, 26): "Second Christmas Day",
+}
```

Comment:
Same as for examples/electricity_demand_example.py.

Reply:
I think it is a good way to let the example run without installing workalendar. For me this has two advantages:

1. Everbody can see how a holiday dict looks like and can adapt it or create a new one
2. The comment explains how it has been created, so that everybody is free to install workalendar and create the dict using it.